config-controller/api/v1alpha1/policy_types.go

```diff
@@ -55,6 +57,15 @@ type SecurityPolicySpec struct {
 	IsDefault          bool                 `json:"isDefault,omitempty"`
 }
 
+// IsValid runs validation checks against the SecurityPolicy spec
+func (p SecurityPolicySpec) IsValid() (bool, error) {
+	if p.IsDefault != false {
```

Comment:
I'm sure we've discussed this, but did we all agree to keep isDefault in the CRD spec?

Reply:
We did for the case where default policies will be CRs, which will be soon.